flottform/server/src/database.spec.ts

```diff
@@ -168,9 +168,68 @@ describe('Flottform database', () => {
 						session: answer,
 						iceCandidates: []
 					})
-			).rejects.toThrow(/peerkey/i);
+			).rejects.toThrow(
+				/clientKey is wrong: Another peer is already connected and you cannot change this info without the correct key anymore. If you lost your key, initiate a new Flottform connection./i
+			);
 			const infoAfter = await db.getEndpoint({ endpointId });
 			expect(infoBefore).toStrictEqual(infoAfter);
 		});
 	});
+
+	describe('startCleanup()', () => {
+		function sleep(ms: number) {
```

Comment:
Just to make totally clear this expects you to wait for it, I would add this:

```suggestion
		async function sleep(ms: number): Promise<void> {
```

But the helper function itself could also go somewhere else and doesn't need to be inside the `describe` block

Reply:
Since I changed to fake timers like you suggested, I don't think we need it anymore!